contracts/TimeManagerV8.sol

```diff
@@ -4,21 +4,24 @@ pragma solidity 0.8.13;
 import { SECONDS_PER_YEAR } from "./constants.sol";
 
 abstract contract TimeManagerV8 {
+    /// @notice Stores blocksPerYear if isTimeBased is true else secondsPerYear is stored
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     uint256 public immutable blocksOrSecondsPerYear;
 
+    /// @notice Acknowledges if a contract is time based or not
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     bool public immutable isTimeBased;
 
+    /// @notice Stores the current block timestamp or block number depending on isTimeBased
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    function() view returns (uint256) private immutable _getCurrentSlot;
+
     /**
      * @dev This empty reserved space is put in place to allow future versions to add new
      * variables without shifting down storage in the inheritance chain
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[48] private __gap;
-
-    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
-    function() view returns (uint256) private immutable _getCurrentSlot;
+    uint256[50] private __gap;
```

Comment:
This contract is already used by the [Prime contract deployed to BNB mainnet](https://bscscan.com/address/0xBbCD063efE506c3D42a0Fa2dB5C08430288C71FC#readProxyContract), source code [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/Tokens/Prime/Prime.sol). Also used by the `PrimeLiquidityProvider` contract.

In both cases, the `TimeManagerV8` contract is at the end of the list of extended contracts, so it shouldn't be a problem to increase the gap. If we do it we should update the version of the npm package used by the core repo asap, to avoid potential issues